src/hotspot/share/runtime/os.hpp

```diff
@@ -223,6 +223,12 @@ class os: AllStatic {
   static bool   pd_unmap_memory(char *addr, size_t bytes);
   static void   pd_free_memory(char *addr, size_t bytes, size_t alignment_hint);
   static void   pd_realign_memory(char *addr, size_t bytes, size_t alignment_hint);
+  static void   pd_pretouch_memory(void *first, void *last, size_t page_size);
+
+  // Some platforms may have special treatments for pretouch, while most
+  // platforms do the same. So the common part of the code was extract here to
+  // avoid copying it around.
```

Comment:
Small grammar fixes:
- treatment in singular
- do the same *thing*
- was extract*ed*
- Missing comma after "So"

```c++
 // Some platforms may have special treatment for pretouch, while most
 // platforms do the same thing. So, the common part of the code was extracted here to
 // avoid copying it around.
```